lib/report.js

```diff
@@ -25,9 +40,34 @@ class Report {
   }
   _getCoverageMapFromAllCoverageFiles () {
     const map = libCoverage.createCoverageMap({})
+    const mergedResults = {}
+    this._loadReports().forEach((report) => {
+      report.result.forEach((result) => {
+        if (this.exclude.shouldInstrument(result.url)) {
+          if (mergedResults[result.url]) {
+            mergedResults[result.url] = v8CoverageMerge(
+              mergedResults[result.url],
+              result
+            )
+          } else {
+            mergedResults[result.url] = result
+          }
+        }
+      })
+    })
 
-    this._loadReports().forEach(function (report) {
-      map.merge(report)
+    Object.keys(mergedResults).forEach((url) => {
+      const result = mergedResults[url]
+      // console.info(JSON.stringify(result, null, 2))
+      try {
+        const path = resolve(this.resolve, result.url)
+        const script = v8toIstanbul(path)
+        script.applyCoverage(result.functions)
+        map.merge(script.toIstanbul())
+      } catch (err) {
+        // most likely this was an internal Node.js library.
+        if (err.code !== 'ENOENT' && err.code !== 'EISDIR') throw err
```

Comment:
@TimothyGu any thoughts about how to easily tell whether the coverage outputted was for one of Node.js' built in libraries, e.g., `internal/inspector`.

Reply:
How about checking whether the URL was an absolute path or not?